packages/adapter-pocketbase/README.md

```diff
@@ -5,11 +5,6 @@
    <p align="center">
    Open Source. Full Stack. Own Your Data.
    </p>
-   <!-- <p align="center" style="align: center;">
-      <img src="https://github.com/nextauthjs/next-auth/actions/workflows/release.yml/badge.svg?branch=main" alt="CI Test" />
-      <a href="https://www.npmjs.com/package/@next-auth/table-storage-adapter" target="_blank"><img src="https://img.shields.io/bundlephobia/minzip/@next-auth/table-storage-adapter" alt="Bundle Size"/></a>
-      <a href="https://www.npmjs.com/package/@next-auth/table-storage-adapter" target="_blank"><img src="https://img.shields.io/npm/v/@next-auth/table-storage-adapter" alt="@next-auth/table-storage-adapter Version" /></a>
-   </p> -->
```

Comment:
Thank you Azure Tables Adapter for serving as an amazing beginner friendly template.